airbyte-webapp/src/components/base/Text/Text.tsx

```diff
@@ -52,16 +52,17 @@ const isHeadingType = (props: TextProps | HeadingProps): props is HeadingProps =
 
 export const Text: React.FC<React.PropsWithChildren<TextProps | HeadingProps>> = React.memo((props) => {
   const isHeading = isHeadingType(props);
-  const { as = "p", centered = false, children } = props;
+  const { as = "p", centered = false, children, className: classNameProp, ...remainingProps } = props;
 
   const className = classNames(
     isHeading
       ? getHeadingClassNames({ centered, size: props.size ?? "md" })
       : getTextClassNames({ centered, size: props.size ?? "md", bold: props.bold ?? false }),
-    props.className
+    classNameProp
   );
 
   return React.createElement(as, {
+    ...remainingProps,
```

Comment:
This was needed so that the `data-id` prop passed in [here](https://github.com/airbytehq/airbyte/pull/16280/files#diff-58fac63dfaf021e0a719b51a3b572abd1b255fcfbec3542ec842eb2f21c8551fR27) is actually set on the resulting element, so that our cypress tests still work